readme.md

```diff
@@ -61,8 +62,24 @@ console.log(dargs(input, {includes: includes}));
 	'--camel-case=5',
 	'--multiple=value',
 	'--multiple=value2',
-	'--sad=:(',
-	'--pie-kind=cherry'
+	'--pie-kind=cherry',
+	'--sad=:('
```

Comment:
You can't switch the order here without switching the `includes` array order too.